bangazon_website/bang_app/templates/categories.html

```diff
@@ -9,6 +9,7 @@
 	<div class="col-md-10 col-md-offset-2">
 		<h3><a href="{% url 'bang_app:product_list' id=category.id %}">{{ category.label }}</a></h3>
 		{% for p in category.products %}
+			<h5>Total Items in Category = ({{p.quantity}})</h5>
```

Comment:
Nice! Can this be refactored to just 
Category Name (Number)?
ie: School Supplies (3)